tools/cellranger.wdl

```diff
@@ -91,7 +91,7 @@ task count {
         cpu: ncpu
         memory: "~{memory_gb} GB"
         disk: "~{disk_size_gb} GB"
-        container: "ghcr.io/stjudecloud/cellranger:1.1.1"
+        container: "docker://ghcr.io/stjudecloud/cellranger:1.1.1"
```

Comment:
```suggestion
        container: 'docker://ghcr.io/stjudecloud/cellranger:1.1.1'
```

For consistency, though the `"` was likely mine.

Reply:
Agree they should all match. But IDK why we use all `"` except for the `container` string. I'd like to make us use double quotes consistently, but that's a more annoying task to perform. I'll probably make it a rule for the future auto-formatter and use that on the repo.

Until then, this should be single quotes to match everywhere else.